File: Solutions/C#/0014-LongestCommonPrefix/LongestCommonPrefix.cs
using System;

namespace LongestCommonPrefix
{
    public class Solution 
    {
        public string LongestCommonPrefix(string[] strs)
        {
            if (strs.Length == 0)
                return "";

            string result = "";
            for (int i = 0; i < strs[0].Length; i++)
            {
                for (int j = 0; j < strs.Length; j++)
                {
                    if (i >= strs[j].Length || strs[j][i] != strs[0][i])
                        return result;
                }

                result += strs[0][i];
            }

            return result;
        }
    }
}

File: Solutions/C++/Strings/LongestCommonPrefix.cpp
#include <vector>
#include <string>

using namespace std;

class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        string res;
        for(int i = 0; i < strs[0].length(); i++) {
            char toCompare = strs[0][i];
            for(int j = 1; j < strs.size(); j++) {
                if(i >= strs[j].length() || strs[j][i] != toCompare )
                    return res;
            }

            res += toCompare;
        }

        return res;
    }
};